packages/vanilla-components/src/components/global-container/global-container.tsx

```diff
@@ -86,10 +86,10 @@ export class GlobalContainer {
         {this.poweredby ? (
           <a
             class="sqh-attribution"
-            href="https://www.saasquatch.com/?utm_source=app&utm_medium=user-widget&utm_campaign=referral-widget"
+            href="https://impact.com/advocate/"
             target="_blank"
           >
-            Powered With Saasquatch By Impact.com
+            Powered By Impact.com
```

Comment:
Shouldn't this be Impact.com/Advocate since that is what is in the logo for mint?